modules/sandboxed-containers-preparing-openshift-cluster.adoc

```diff
@@ -2,34 +2,35 @@
 //
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
-[id="sandboxed-containers-preparing-openshift-cluster_{context}"]
-= Preparing your cluster for {sandboxed-containers-first}
+[id="sandboxed-containers-prerequisites_{context}"]
+= Prerequisites
 
 Before you install {sandboxed-containers-first}, ensure that your {product-title} cluster meets the following requirements:
 
-* Your cluster must be installed on bare metal infrastructure with {op-system-first} workers. Your cluster must use installer-provisioned infrastructure.
-// To Do: Add link to bare metal instructions: ../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[bare metal]
-// To Do: Add link to installer provisioned infrastructure: ../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[Installer provisioned infrastructure]
+* Your cluster must be xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[installed on bare metal infrastructure] with {op-system-first} workers.
```

Comment:
Xrefs shouldn't be used in modules (for more info, see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references)

Reply:
Oh gosh, I didn't even realize this was here. It's a remnant from before I got it :/ Moving it to the assembly in an "Additional resources".